### PR TITLE
Update collection-index.yml

### PR DESCRIPTION
**Pull request by shepherdjerred**
Update with my features repository: https://github.com/shepherdjerred/devcontainers-features

Currently, the only feature is to install [Earthly](https://earthly.dev/)

Also, shout-out to the author(s) of the starter template. I went in with zero knowledge and had a working feature complete with tests in about an hour!

### CODE REVIEW

File: _data/collection-index.yml [hunk 1]
```diff
@@ -438,3 +438,8 @@
   contact: https://github.com/nullcoder/devcontainer-features/issues
   repository: https://github.com/nullcoder/devcontainer-features
   ociReference: ghcr.io/nullcoder/devcontainer-features
+- name: Features by shepherdjerred
+  maintainer: Jerred Shepherd
+  contact: https://github.com/shepherdjerred/devcontainers-features/issues
+  repository: https://github.com/shepherdjerred/devcontainers-features
+  ociReference: ghcr.io/shepherdjerred/devcontainers-features
```
Comment (samruddhikhandale) [inline _data/collection-index.yml:445]: Hi 👋

Can you help change the visibility of this metadata package (and your Feature package) to public? The [index](https://containers.dev/features) won't be able to crawl the Features in `shepherdjerred/devcontainers-features` if it's private.

More details - https://containers.dev/guide/author-a-feature#publishing

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]

File: _data/collection-index.yml (hunk 1 above)
Comment (shepherdjerred) [reply _data/collection-index.yml:445]: Hi! I believe it already is public

CleanShot 2023-06-12 at 10 31 42@2x
CleanShot 2023-06-12 at 10 32 01@2x

Comment (samruddhikhandale) [reply _data/collection-index.yml:445]: Gotcha, thanks for double checking.
Apparently, my oras login had expired which lead to 403s, my bad!

**Review by samruddhikhandale — APPROVED** [review]
Thank you! ✨

File: _data/collection-index.yml (hunk 1 above)
Comment (shepherdjerred) [reply _data/collection-index.yml:445]: No worries at all!

**Pull request merged**